Create README.md (#27)
ebpffoundation/bsc: README.md

* **Coordinating collaboration** for eBPF, ensuring alignment between tools, libraries, and projects, and fostering integration across various use cases.
* **Coordinating and evangelizing eBPF** projects and technologies to current and potential developers and users.
* **Identifying areas of collaboration** with adjacent open source communities (e.g., kernel, compiler, and debugger communities) to encourage the widespread adoption and integration of eBPF technology.
* **Proposing priorities** to the Governing Board for community developmen actives, events, travel funds, and other investments that will foster growth in the eBPF developer and user communities.

By facilitating collaboration and transparency, the eBPF Foundation BSC strives to guide the long-term success of eBPF for all.

## **BSC Technical Vision**

The technical direction of the ecosystem is problem-centric. We encourage and support eBPF to solve problems faced by adopters, to build the right ecosystem for innovation, and embrace changes in the broader technical landscape. To exercise and increase the expertise within the technical community to compose solutions that stand the test of time.

## **Members**

> Andrii Nakryiko, Meta (2021-10-13 to present)
> Brendan Gregg, Intel (2021-10-13 to present)
> Daniel Borkmann, Isovalent (2021-10-13 to present)
> KP Singh, Google (2021-10-13 to present)
> Joe Stringer, Isovalent (2021-10-13 to present)
> Alexei Starovoitov, Meta (2021-10-13 to present)
> Alan Jowett, Microsoft (2023-11-29 to present)

Author: ```suggestion
Andrii Nakryiko, Meta (2021-10-13 to present)

Brendan Gregg, Intel (2021-10-13 to present)

Daniel Borkmann, Isovalent (2021-10-13 to present)

KP Singh, Google (2021-10-13 to present)

Joe Stringer, Isovalent (2021-10-13 to present)

Alexei Starovoitov, Meta (2021-10-13 to present)

Alan Jowett, Microsoft (2023-11-29 to present)
```

## **eBPF Steering Committee (BSC)**

The eBPF Foundation eBPF Steering Committee (BSC) is the technical and collaborative governing body of the eBPF community as defined in the Foundation [charter](https://ebpf.foundation/charter/). Our mission is to advance the development, adoption, and best practices of eBPF technology through consensus-driven, community-led initiatives. We are responsible for:

* **Defining and maintaining the technical vision** for the eBPF ecosystem, ensuring it aligns with industry needs and innovations.

> * **Creating a cohesive architecture** for eBPF, ensuring alignment between tools,libraries, and projects, and fostering integration across various use cases.

Author: ```suggestion
* **Coordinating collaboration** for eBPF, ensuring alignment between tools, libraries, and projects, and fostering integration across various use cases.
```

* **Collecting and acting on feedback** from users, contributors, and industry leaders and using it to inform the evolution of eBPF and community standards.

> * **Standardizing interfaces** between components, where necessary, by aligning on best practices and implementations.

Author: ```suggestion
```

> * **Encouraging and promoting common practices** across the eBPF ecosystem to ensure consistency, reliability, and ease of adoption.

Author: ```suggestion
```

* **Identifying areas of collaboration** with adjacent open source communities (e.g., kernel, compiler, and debugger communities) to encourage the widespread adoption and integration of eBPF technology.

> * **Proposing priorities** to the Governing Board for events, travel funds, and other investments that will foster growth in the eBPF developer and user communities.

Author: ```suggestion
* **Proposing priorities** to the Governing Board for community developmen actives, events, travel funds, and other investments that will foster growth in the eBPF developer and user communities.
```

* **Defining and maintaining the technical vision** for the eBPF ecosystem, ensuring it aligns with industry needs and innovations.
* **Creating a cohesive architecture** for eBPF, ensuring alignment between tools,libraries, and projects, and fostering integration across various use cases.
* **Collecting and acting on feedback** from users, contributors, and industry leaders and using it to inform the evolution of eBPF and community standards.
* **Standardizing interfaces** between components, where necessary, by aligning on best practices and implementations.
* **Encouraging and promoting common practices** across the eBPF ecosystem to ensure consistency, reliability, and ease of adoption.
* **Identifying areas of collaboration** with adjacent open source communities (e.g., kernel, compiler, and debugger communities) to encourage the widespread adoption and integration of eBPF technology.

> * **Proposing priorities** to the Governing Board for events, travel funds, and other investments that will foster growth in the eBPF developer and user communities.

Reviewer: These seem like they go a little further than the charter declares, is that intentional?

Author: I think the charter is a bit vague by saying "coordinating collaboration within the eBPF community". I tried to flesh out what that could actually mean in reality and I think those points fall under "coordinating collaboration", but I can also cut it back if you prefer.

Author: it has been cut back now

Reviewer: I noticed that yeah. My original thought on this thread is that the BSC sat down and discussed at length about the scope of responsibilities they wanted to establish within the eBPF Foundation, and the list of items in the charter section 4.2 was the result. Hopefully that list should be practical enough for anyone interacting with the eBPF Foundation / BSC, though I recognize your feedback that the coordination could be further clarified. As long as this text accurately reflects the same scope of the charter then this makes sense. Shortest path is copying 1-to-1 with the charter text, but looking at the proposed text currently I think that this PR accurately reflects the text in the charter.

## **BSC Technical Vision**

The technical direction of the ecosystem is problem-centric. We encourage and support eBPF to solve problems faced by adopters, to build the right ecosystem for innovation, and embrace changes in the broader technical landscape. To exercise and increase the expertise within the technical community to compose solutions that stand the test of time.

## **Members**

Andrii Nakryiko, Meta (2021-10-13 to present)
Brendan Gregg, Intel (2021-10-13 to present)
Daniel Borkmann, Isovalent (2021-10-13 to present)
KP Singh, Google (2021-10-13 to present)
Joe Stringer, Isovalent (2021-10-13 to present)
Alexei Starovoitov, Meta (2021-10-13 to present)
Alan Jowett, Microsoft (2023-11-29 to present)

**Emeritus**

Lorenz Bauer (2021-10-13 to 2023-02-08)
Dave Thaler (2021-10-13 to 2023-10-18)

## **Membership and Voting**

Any BSC member and/or the Governing Board can propose a new member to be added to the BSC. To accept a new member, two-thirds of the entire BSC must approve. Any member of the BSC may be removed by a two-thirds majority of the other BSC members.

> Ideally, decisions are made by consensus, but if a vote is needed, each member gets one vote, and most decisions are made by a simple majority. For BSC meetings, at least 60% of voting members must be present to make decisions. If not enough members are present, the meeting can continue but no decisions can be made.

Reviewer: Do we need to duplicate this from the charter to the readme?

Author: I thought it would be helpful to include here in simpler language because I don't expect most people to read the charter, but I do think they want to know how decisions are made.

## **Meeting Time and Minutes**

The BSC meets roughly every two weeks and all meeting minutes can be found in the Github repo.

## **Contact Us**

Email: bsc at lists.ebpf.foundation

> If you want to add a topic or speak at one of the next BSC meetings, please submit the [form on the website](https://ebpf.foundation/participate/). A BSC committee member will reach out to you if they add it to the agenda.

Reviewer: Might be worth doing a test of this form. I don't remember receiving any such requests. I wouldn't want someone to submit something and get zero feedback.

Author: I'll remove for now

> 
> If you want to add a topic or speak at one of the next BSC meetings, please submit the [form on the website](https://ebpf.foundation/participate/). A BSC committee member will reach out to you if they add it to the agenda.

Author: ```suggestion
```

* **Coordinating collaboration** for eBPF, ensuring alignment between tools, libraries, and projects, and fostering integration across various use cases.
* **Coordinating and evangelizing eBPF** projects and technologies to current and potential developers and users.
* **Identifying areas of collaboration** with adjacent open source communities (e.g., kernel, compiler, and debugger communities) to encourage the widespread adoption and integration of eBPF technology.
* **Proposing priorities** to the Governing Board for community developmen actives, events, travel funds, and other investments that will foster growth in the eBPF developer and user communities.

By facilitating collaboration and transparency, the eBPF Foundation BSC strives to guide the long-term success of eBPF for all.

## **BSC Technical Vision**

The technical direction of the ecosystem is problem-centric. We encourage and support eBPF to solve problems faced by adopters, to build the right ecosystem for innovation, and embrace changes in the broader technical landscape. To exercise and increase the expertise within the technical community to compose solutions that stand the test of time.

## **Members**

> Andrii Nakryiko, Meta (2021-10-13 to present)

Reviewer: Andrii stepped down on 2024-10-30.

> Andrii Nakryiko, Meta (2021-10-13 to present)
> 

Author: ```suggestion
```

Brendan Gregg, Intel (2021-10-13 to present)

Daniel Borkmann, Isovalent (2021-10-13 to present)

KP Singh, Google (2021-10-13 to present)

Joe Stringer, Isovalent (2021-10-13 to present)

Alexei Starovoitov, Meta (2021-10-13 to present)

Alan Jowett, Microsoft (2023-11-29 to present)

**Emeritus**

> Lorenz Bauer (2021-10-13 to 2023-02-08)
> Dave Thaler (2021-10-13 to 2023-10-18)
> 

Author: ```suggestion
Lorenz Bauer (2021-10-13 to 2023-02-08)

Dave Thaler (2021-10-13 to 2023-10-18)

Andrii Nakryiko, Meta (2021-10-13 to 2024-10-30)
```
